### PR TITLE
Increase element wait timeout for screenshot tests

### PR DESCRIPTION
**Pull request by shifucun**
Removed duplicated ASYNC_ELEMENT_CLASS tagging, which exist twice in ranking tile.

### CODE REVIEW

File: static/js/components/tiles/sv_ranking_units.tsx [hunk 1]
```diff
@@ -100,9 +99,7 @@ export function SvRankingUnits(props: SvRankingUnitsProps): JSX.Element {
   return (
     <React.Fragment>
       {rankingMetadata.showHighestLowest || props.errorMsg ? (
-        <div
-          className={`ranking-unit-container ${ASYNC_ELEMENT_CLASS} highest-ranking-container`}
```
Comment (keyurva) [inline static/js/components/tiles/sv_ranking_units.tsx:104]: Do the webdriver tests run without issues with this change?

File: static/js/components/tiles/sv_ranking_units.tsx [hunk 2]
```diff
@@ -21,7 +21,6 @@ import React, { RefObject, useRef } from "react";
 
 import { VisType } from "../../apps/visualization/vis_type_configs";
 import { URL_PATH } from "../../constants/app/visualization_constants";
-import { ASYNC_ELEMENT_CLASS } from "../../constants/css_constants";
```
Comment (pradh) [inline static/js/components/tiles/sv_ranking_units.tsx:24]: For my understanding, why remove this?  There are other places that use it too.

**Review by pradh — APPROVED** [review]
Thanks!

File: static/js/components/tiles/sv_ranking_units.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/components/tiles/sv_ranking_units.tsx:104]: This is a dup. The child html also have this class added and should be preferred.

File: static/js/components/tiles/sv_ranking_units.tsx (hunk 2 above)
Comment (shifucun) [reply static/js/components/tiles/sv_ranking_units.tsx:24]: ASYNC_ELEMENT_CLASS in this file is a dup. Wherever it's used, the child html also have this class added and should be preferred.

**Pull request merged**